deps.bzl

```diff
@@ -7078,6 +7078,14 @@ def install_buildbuddy_dependencies(workspace_name = "buildbuddy"):
         downloaded_file_path = "soci-store-v0.0.8-linux-amd64",
     )
 
+    http_file(
+        name = "com_github_buildbuddy_io_soci_snapshotter-soci-store-v0.0.9-linux-amd64-race",
```

Comment:
remove the version from the rule & downloaded file name to make your life easier

Reply:
Good call, thanks!